resources/targets/simple.yaml

```diff
@@ -1,3 +1,7 @@
+-
+    name: Kepler 488 b
+    position: 18:51:51.88s +48:20:42.3s
```

Comment:
1) this file shouldn't be in this PR. And yes, we need a better way for targets to be changed. :)
2) I would be surprised if this parses in this format, with the `:` instead of the `hms`

Reply:
Sorry, this wasn't supposed to be part of the change.
And you're right, this format wouldn't parse. OTOH, it would be nice if it did since that is what some sites present for the coords.